creator-node/src/components/replicaSet/replicaSetController.js

```diff
@@ -126,9 +128,11 @@ const syncRouteController = async (req, res) => {
           wallet
         },
         forceWipe: req.body.forceWipe,
-        logContext: req.logContext
+        logContext: req.logContext,
+        parentSpanContext: tracing.currentSpanContext()
```

Comment:
when do we need parentSpanContext?

Reply:
https://github.com/AudiusProject/audius-protocol/blob/fa08aa57813c2658c1ce693ff7987a5b1ae6fe0e/creator-node/src/services/sync/syncImmediateQueue.js#L52 

That's what gets linked when the job runs